src/ActionMenu/ActionMenu.tsx

```diff
@@ -89,7 +89,12 @@ type MenuOverlayProps = Partial<OverlayProps> &
      */
     children: React.ReactNode
   }
-const Overlay: React.FC<React.PropsWithChildren<MenuOverlayProps>> = ({children, align = 'start', ...overlayProps}) => {
+const Overlay: React.FC<React.PropsWithChildren<MenuOverlayProps>> = ({
+  children,
+  align = 'start',
+  'aria-labelledby': ariaLabelledby,
```

Comment:
This was the only change that needed to be pulled in from the deleted file. (added in https://github.com/primer/react/pull/1759)